client/store/types/actions/locations.ts

```diff
@@ -0,0 +1,19 @@
+export const FETCH_START = 'fcc/chapter/LOCATIONS_START';
```

Comment:
these probably aren't necessary with typescript, but it doesn't matter for now

Reply:
Why not? They're simpler to use because you can easily autocomplete them + less to type